tests/steps/steps.py

```diff
@@ -1,28 +1,42 @@
-from behave import given, when, then
 import base64
-from algosdk import kmd
-from algosdk.future import transaction
-from algosdk import encoding
-from algosdk import algod
-from algosdk import account
-from algosdk import mnemonic
-from algosdk import wallet
-from algosdk import auction
-from algosdk import util
-from algosdk import constants
-from algosdk import logic
-from algosdk.future import template
 import os
+import random
 from datetime import datetime
-import hashlib
 
+from algosdk import (
```

Comment:
Definitely an improvement in making imports from `algosdk` more pythonic. However, I think the external imports from `behave` and `nacl.signing` ought to be put in a section above

Reply:
I used the auto-import on VS Code - I think your suggestion makes sense but might require more manual setup/overriding some defaults on the editor.